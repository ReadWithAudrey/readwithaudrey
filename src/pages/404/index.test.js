import React from 'react'
import renderer from 'react-test-renderer'
import NotFoundPage from './index.js'

describe('NotFoundPage page snapshot test', () =>
  it('renders correctly', () => {
    const tree = renderer.create(<NotFoundPage />).toJSON()
    expect(tree).toMatchSnapshot()
  }))
