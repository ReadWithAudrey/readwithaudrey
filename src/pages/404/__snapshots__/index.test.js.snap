// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFoundPage page snapshot test renders correctly 1`] = `
Array [
  <header
    className="sc-bdVaJa flex items-center justify-between ph3 montserrat white bg-green w-100 f3 chiVga"
  >
    <div
      className="flex items-center"
      id="header-left"
    >
      <img
        alt="book icon"
        className="h2 ph2"
        src="test-file-stub"
      />
      <Link
        to="/"
      >
        Audrey
      </Link>
    </div>
    <Link
      className="w-third"
      to="/Form1/"
    >
      <button
        className="sc-gzVnrw bg-green hover-bg-pink pa2 mv2 white br2 w-100 f5 montserrat iQUxnM"
      >
        Join Us
      </button>
    </Link>
  </header>,
  <div
    className="flex flex-column items-center justify-center page-padding ph4"
  >
    <h1>
      NOT FOUND
    </h1>
    <p>
      You just hit a route that doesn't exist... the sadness.
    </p>
  </div>,
  <footer
    className="sc-bwzfXH flex flex-column justify-center items-center montserrat bg-green w-100 bFHdpU"
  >
    <span
      className="sc-htpNat flex items-center justify-around bg-green w-100 jbutSu"
    >
      <Link
        className="no-underline white ph1"
        to="/"
      >
        Home
      </Link>
      <Link
        className="no-underline white ph1"
        to="/story/"
      >
        Our Story
      </Link>
      <Link
        className="no-underline white ph1"
        to="/faq/"
      >
        FAQs
      </Link>
      <Link
        className="no-underline white ph1"
        to="/contact/"
      >
        Contact Us
      </Link>
    </span>
    <span
      className="sc-bxivhb flex justify-around items-center bg-white black w-100 cHkvlR"
    >
      Â© Audrey
      <Link
        className="no-underline black ph1"
        to="/Terms/"
      >
        Terms
      </Link>
      <Link
        className="no-underline black ph1"
        to="/Privacy/"
      >
        Privacy
      </Link>
    </span>
  </footer>,
]
`;
